.gitignore

```diff
@@ -1,2 +1,2 @@
 node_modules
-/config/config.json
\ No newline at end of file
+/config/config.json
```

Comment:
recommend adding empty final line for linux system compatibility. same goes for other files